### ISSUE TITLE
Participants has trouble opening Keyboard Shortcuts dialog 

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#127]
In two recent interviews, both users had trouble opening the Keyboard Shortcuts dialog. I am not sure why this is as it opens fine for me when using Safari and Voice Over.

I'll collect the dev links that were used and the AT that was used int eh interviews.

**Comment by terracoda** [comment]
@briannatomlinson, I emailed you and @pkaini, too about this issue.

Please assign to @KatieWoe for re-assignment once you have added any details you have.

**Comment by terracoda** [comment]
I added the help wanted label to alert the fact that I will need the QA team help to verify the problem defined in this issue.

**Comment by terracoda** [comment]
@briannatomlinson, on second thought, can you and @pkaini try and reporduce the issue and add details to this issue, so we can try and figure out why the Keyboard Shortcuts dialog did not open and if there is something we need to fix.

You can reassign the issue to me when we have more information about the problem.

**Comment by briannatomlinson** [comment]
sure!

**Comment by pkaini** [comment]
Firefox 62.0
NVDA 2017.4
OS (n/a)
Link to sonified described version: sim (https://bayes.colorado.edu/dev/html/phet-io-wrapper-sonification/1.0.0-dev.75/phet-io-wrapper-sonification/friction/html/friction-sonification.html?sonificationFile=sonificationDefault&accessibility)

didn't have trouble opening keyboard shortcut menu but when on it, screen reader read "dialog blank" but it also read the contents in the menu

Second Interview (Sept 24, 2019):

Firefox 61.0
NVDA (from March) 2018.1
OS (n/a)
Link to sonified described version: same as above

not able to open the keyboard shortcuts

@briannatomlinson @terracoda

**Comment by terracoda** [comment]
@pkaini, thanks for this.

Yes, I am aware of the issue you describe above, i.e., both labels for the buttons are read aloud when focus is on the first button. This is a definitely an issue.

However, it also seemed like the participants, or maybe just one of them, could not even open the Keyboard Shortcuts dialog. In the recording, you actually say, "I think it was disabled".

Could you try to reproduce that issue? It is not a problem for me with Voice Over and Safari.

**Comment by briannatomlinson** [comment]
@terracoda I tried to reproduce it, but couldn't really get it. It's not clear to me if they're just not hitting the right keys, or what. In one of them, it seems like she hits "a" (like to get annotations) but it doesn't have one associated with it.

So I don't know if they don't know how to open it when they get there (like we've had to tell people how to open it). When they get to the "sim resources and tools" and the "button keyboard shortcuts" they don't know how to open it.

It's like some people get down there and don't realize what it's supposed to do

**Comment by terracoda** [comment]
@briannatomlinson, thanks. This is interesting.
It could be that because a lot of stuff is read out at once, they do not recognize that they are on a simple pop-up button. They become unsure what to do - they likely want to investigate the Keyboard Shortcuts, but the last thing they hear is P h E T menu.

It may just be confusion.
The resource bar is slated to be restructured. Hopefully. that will resolve the issue.

In interviews up until this is fixed, please watch for trouble here, and let the users know that there is a know issue, but they are on a button.

**Comment by terracoda** [comment]
Also, in the new structure, there is on-demand "help text" associated with these buttons (https://github.com/phetsims/joist/issues/523#issuecomment-424537326). Having a content element in between the buttons, or even just another wrapping `div` will likely prevent NVDA from reading the button labels out back-to-back when only one the first one has focus.
